0XXX-verification-mapping

```diff
@@ -0,0 +1,97 @@
+# HIP XXX: Verification Mapping for MOBILE Network
+
+- Author: @zer0tweets, @jpad
```

Comment:
it would save the hip editors / foundation time if these sections matched existing HIPs before a merge.

Reply:
on it